test/sketcherBasicSMILES.h

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
why the prefix "sketcher" in this file's name? Coordgen used to prefix everything "sketcherMinimizer" as you can tragically still see in many files that were not refactored, but I think anything new we add should use the prefix "Coordgen"

Reply:
good call. Updating.

Reply:
thanks, good call.